lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -152,13 +152,22 @@ def ss_unknown_no_embassies?(country_slug)
     def os_consular_cni_in_nearby_country?(country_slug)
       OS_CONSULAR_CNI_IN_NEARBY_COUNTRY.include?(country_slug)
     end
-    
+
+    def countries_with_18_outcomes
+      countries = marriage_data.fetch(:countries_with_18_outcomes)
+      valid_outcomes_country_data_structure?(countries) ? countries : []
```

Comment:
I think raising an exception might be better here if the data is invalid? Otherwise if something goes wrong, a valid, albeit probably wrong value of `[]` will be returned.